index.html

```diff
@@ -353,6 +354,14 @@
             </li>
           </ul>
         </section>
+        <section id="httpGETLDPRS">
+          <h2><a>LDP-RS</a>s</h2>
+          <p>
+            <code>GET</code> requests that include a <code>Prefer</code> header to any <a>LDP-RS</a> MUST
```

Comment:
That is a helpful refinement, @zimeon. Changing:
> `GET` requests that include a `Prefer` header...

to:
> Responses to `GET` requests that apply a `Prefer` request header...

..since, regardless of whether the client's `GET` includes the `Prefer` header, this section of the specification only applies if a server applies the `Prefer` header.